src/main/java/org/cactoos/text/Sticky.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Scalar;
+import org.cactoos.Text;
+
+/**
+ * Cached version of a Text.
+ *
+ * <p>This {@link Text} decorator technically is an in-memory
+ * cache.</p>
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @see org.cactoos.scalar.Sticky
+ * @since 0.47
+ */
+public final class Sticky implements Text {
+    /**
+     * Sticky scalar.
+     */
+    private final Scalar<String> scalar;
+
+    /**
+     * Ctor.
+     * @param txt Text to cache
+     */
+    public Sticky(final Text txt) {
+        this((Scalar<String>) txt::asString);
+    }
+
+    /**
+     * Private ctor.
+     * @param sclr Scalar to cache
+     */
+    private Sticky(final Scalar<String> sclr) {
```

Comment:
@andreoss please remove this constructor and just keep the `Text` one

Reply:
@victornoel Done